05-profile-specification.bs

```diff
@@ -0,0 +1,76 @@
+<pre class='metadata'>
+Title: TREE Profile specification
+Shortname: TREEProfile
+Level: 1
+Status: w3c/CG-DRAFT
+Markup Shorthands: markdown yes
+URL: https://w3id.org/tree/specification/profile
+Repository: https://github.com/treecg/specification
+Mailing List: public-treecg@w3.org
+Mailing List Archives: https://lists.w3.org/Archives/Public/public-treecg/
+Editor: Pieter Colpaert, https://pietercolpaert.be
+Abstract:
+    This specification defines a profile to allow clients to parse in a streaming fashion and extract TREE members as a replacement of the more computationally intensive Member Extraction Algorithm.
+</pre>
+
+
+# Introduction # {#introduction}
+
+This specification introduces [profiles](https://profilenegotiation.github.io/I-D-Accept--Schema/I-D-accept-schema) for serializations such as n-triples, turtle and TRiG to first contain a group for the hypermedia, and then contain groups for the members.
+The profile is optional to implement, yet comes with the benefit that it allows TREE clients to speed up parsing of individual members and process them in a streaming fashion.
```

Comment:
Minor, but I would change `TREE clients to speed up parsing` -> `TREE clients to speed up extracting` as the actual parsing of the document takes the same amount of time, parsing is mapping text to quads.

Reply:
Agreed!